serializer_impl.hh

```diff
@@ -321,6 +322,25 @@ struct serializer<utils::small_vector<T, N>>
     : idl::serializers::internal::vector_serializer<utils::small_vector<T, N>>
 { };
 
+template<>
+struct serializer<service::mixed_change> {
```

Comment:
You shouldn't have to implement serializer manually. See how it's handled for `schema_change` or `topology_change` and do the same. (See idl/ directory)

Reply:
I tried, but eventually ended up writing this custom one. The thing is, my `mixed_change` struct is different from `xxx_change` and apparently this lib cannot automatically generate a serializer for it.

Reply:
Can I close this comment?

Reply:
> The thing is, my mixed_change struct is different from xxx_change and apparently this lib cannot automatically generate a serializer for it.

Please investigate why. It should be able to.

Reply:
Removed altogether, thanks.